src/services/api/features/package-actions/handlers/withdraw-package-lambda.ts

```diff
@@ -0,0 +1,17 @@
+import { APIGatewayEvent } from "aws-lambda";
+import { handleEvent } from "../lambda-handler";
+import { z } from "zod";
+import { response } from "@/shared/lambda-response";
+
+export const withdrawPackageLambda = async (event: APIGatewayEvent) => {
+  return await handleEvent({
+    event,
+    schema: z.object({}),
+    async lambda(data) {
+      return response({
```

Comment:
Is this missing the actual action line, or does that somehow get covered elsewhere?